pkg/volume/quobyte/quobyte.go

```diff
@@ -103,7 +102,8 @@ func (plugin *quobytePlugin) CanSupport(spec *volume.Spec) bool {
 		glog.V(4).Infof("quobyte: Error: %v", err)
 	}
 
-	if out, err := exec.New().Command("ls", "/sbin/mount.quobyte").CombinedOutput(); err == nil {
+	exec := plugin.host.GetExec(plugin.GetPluginName())
```

Comment:
Ideally the plugin would not have to pass his own name, but each plugin would get a private "host" struct with the name and things baked in.